WatchFaceFormat/validator-plugin/plugins/src/main/kotlin/MemoryFootprintTask.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.android.build.api.variant.BuiltArtifactsLoader
+import org.gradle.api.DefaultTask
+import org.gradle.api.GradleException
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputDirectory
+import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.PathSensitive
+import org.gradle.api.tasks.PathSensitivity
+import org.gradle.api.tasks.TaskAction
+import org.gradle.process.ExecOperations
+import java.io.File
+import javax.inject.Inject
+
+/**
+ * Runs the Memory Footprint checker tool.
+ */
+abstract class MemoryFootprintTask @Inject constructor(@Internal val execOperations: ExecOperations) :
+    DefaultTask() {
+    init {
+        this.outputs.upToDateWhen { true }
```

Comment:
Worth a comment about why this is required?

https://github.com/liutikas/gradle-best-practices/blob/d7f6d6d8c8924c020b2287c958a76aa1f59eacf1/README.md?plain=1#L148

Reply:
Added.